packages/components/src/Heading/Heading.tsx

```diff
@@ -22,16 +22,26 @@ type Props = {
   weight?: TypographyProps<HeadingElement>["weight"];
 };
 
-const Heading = forwardRef<HTMLElement, Props>(({ as, children, size, /**
-   * Components that wrap typography sometimes require props such as
-   * event handlers, tabIndex, etc. to be passed down into the element.
-   * TODO: add better typing or documentation for optional props,
-   * e.g. React.HTMLAttributes<HTMLHeadingElement>
-   */ ...rest }: Props, ref) => (
-  <Typography as={as || size} size={size} ref={ref} {...rest}>
-    {children}
-  </Typography>
-));
+const Heading = forwardRef<HTMLElement, Props>(
```

Comment:
Eslint/Prettier fixed up this file (and a few others). No idea why this either didn't happen with NPM, or is happening differently.

Reply:
The [prettier online playground agrees with the changes made in this PR](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAJOBDAEwEsoBzTAXkwDMIAnAdzzoICU5qAKT4TPNADSYwACyIAbAnQRC0RAF5whAegBUqgDpRMO1ZgDCEALYAHaAhhpMMEXiwM6eE9YCeZ0o5MiXmNMbgwREZwVtIAjgCuRNKYJnQQJlZoEaJ8aFo6mHpwAG4WmLZQBOJwdILWeABGAJJFcAAeQgFgAHTWEJiVcLH8aHAEmAQQDNokMB023XAlwbAtGbqYACoA8gAiK0h8BANdMPB0riYk5PSDEGARszB2RNA0ZwmB0HjisfGJAgtZmHAtpG12HgwDAWtglgBZAAyAEF9nQiJUIvA0AAecHQ3CEE4AURmFgAfN9VMpMC1ydIMJgAL5bAAKH3K0moAEpKATMJwFqilm4IB4nN40hRgPxMAAfcW+BRwanSxQiuSKOXMkXMuXAcktSkwalE7SZYCiCRSBDU7nKXnuTzefUslkAbhAAhATzuUDQyFAzHiDDpzAQnpQryYLk9LsqjjAAGsAgBlEzAk7IGB0CJKEBwIxdHb9KF4MgRPCkOAAMXoRjsgTIyBAeGREGdIBEMCM4gA6mIUYmwHA44GiIEcoOXLWwGhwyASH06DAGcXK8hqK8+i6AFZoeoAISjsZgcbwwShJDgS5XGY39TjJxKAEUIhB4GfxKuQImyqVa5UqtMm3Exu2RAEDYyAABwAAwunEEB9O2ni1nEISlHkTaRI+cAMgkQZ1mgAC0UBwP0-RNuEUTSPOpCLkgy4vhmfRGEQKZpnRN5wPe6HPq+NyVIBwEiMgABMLqpngEgnIYRhUZmaAAKxNhEfRLFUQY0a+OTprUBAWHGYAIiYMAwkUcYwC4JScbK1JAA).

Reply:
Actually, I bet when installing the deps through yarn we ended up with an updated version of Prettier, and they may have tweaked their formatting.

Reply:
that would make sense 🤷‍♀️